tools/pythonpkg/src/pyconnection.cpp

```diff
@@ -450,7 +470,6 @@ unique_ptr<QueryResult> DuckDBPyConnection::ExecuteInternal(const string &query,
 		py::gil_scoped_release release;
 		unique_lock<std::mutex> lock(py_connection_lock);
 
-		auto statements = connection->ExtractStatements(query);
```

Comment:
This is causing issues, we now use ParseQuery instead of ExtractStatements
Which means pragmas don't get resolved, causing some existing tests to fail